### PR TITLE
Replace `run-local-android` recipe's LOCAL_IP variable

### LINKED ISSUE #989: Run-local-android doesn't work on linux ubuntu
As mentioned in #901, on my machine, `just run-local-android` outputs

```
Target file "--dart-define=ESPLORA_ENDPOINT=http://10.5.0.1
172.20.0.1
172.17.0.1
192.168.100.157
192.168.122.1:3000" not found.
```

This is because the `grep` commands for `LOCAL_IP` output a whole list of local IPs, some of which are internal, broadcast addresses, or docker addresses.

### PR DESCRIPTION
**Pull request by Restioson**
This uses a combination of answers in https://stackoverflow.com/a/25851186 and https://stackoverflow.com/a/49552792.

Fixes #989 (for me). @da-kami, can you confirm that this still works on OS X?

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -80,7 +80,7 @@ run-regtest args="":
 
 run-local-android args="":
     #!/usr/bin/env bash
-    LOCAL_IP=$(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1')
+    LOCAL_IP=$(ip -o route get to 1 | sed -n 's/.*src \([0-9.]\+\).*/\1/p')
```
Comment (bonomat) [inline justfile:83]: `ip` is unfortunately not installed on a mac :( 
If this works for you, maybe we do a check for the operating system and execute `ip` for `linux` and `ifconfig` for `mac`?

Comment (bonomat) [reply justfile:83]: E.g. with this it should be relatively easy: https://github.com/casey/just#enabling-and-disabling-recipes

Comment (klochowicz) [reply justfile:83]: the simplest way to check in a bash script would be checking by calling `uname` from a script. on macs, it returns `Darwin`, so you can match that for mac branch.

Comment (Restioson) [reply justfile:83]: `just` has a function for this https://github.com/casey/just#system-information

**Comment by Restioson** [comment]
Oh no, I've cursed this branch

**Review by da-kami — APPROVED** [review]
works on my machine!

**Pull request merged**